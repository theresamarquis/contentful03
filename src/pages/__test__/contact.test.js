import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '../contact';

describe('Contact', ()=>{
	it('renders correctly', ()=>{
		const tree = renderer.create(<Contact />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})