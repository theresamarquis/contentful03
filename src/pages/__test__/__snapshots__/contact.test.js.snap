// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly 1`] = `
<div>
  <Link
    to="/"
  >
    Home
  </Link>
  <form>
    <label>
      Email:
      <input
        onChange={[Function]}
        placeholder="email"
      />
    </label>
    <input
      type="submit"
      value="Submit"
    />
  </form>
</div>
`;
